### PR TITLE
Configure Magento 2 Cron

### PR DESCRIPTION
**Pull request by kierenevans**
* Configure Magento 2 Cron, but not in the development environment as it will slow down machines
* Fix bug that stops the environment starting when cron is in the services list
* Build and publish cron docker image if cron is in the services list
* Fix cron environment variables being treated as one string

### CODE REVIEW

File: src/_base/harness/scripts/enable.sh.twig [hunk 1]
```diff
@@ -43,22 +43,37 @@ dynamic()
         passthru "docker-compose config --services | grep -v php-fpm | xargs docker-compose pull"
         passthru "docker-compose config --services | grep -v cron | xargs docker-compose build --pull"
         passthru docker-compose build cron
+        # Bring up all but cron
+        passthru "docker-compose config --services | grep -v cron | xargs docker-compose up -d"
```
Comment (tkotosz) [inline src/_base/harness/scripts/enable.sh.twig:47]: noice!

File: src/_base/harness/scripts/enable.sh.twig [hunk 2]
```diff
@@ -43,22 +43,37 @@ dynamic()
...
+    {% else %}
+        # Bring up all services
+        passthru docker-compose up -d
+    {% endif %}
+
     passthru docker-compose exec -T -u build console app init
+    # Bring up all services
+    passthru docker-compose up -d
```
Comment (tkotosz) [inline src/_base/harness/scripts/enable.sh.twig:76]: hmm is this needed here?

**Review by tkotosz — APPROVED** [review]
left a question but other than that it looks good to me

File: src/_base/harness/scripts/enable.sh.twig (hunk 2 above)
Comment (kierenevans) [reply src/_base/harness/scripts/enable.sh.twig:76]: Brings up the cron which didn't get started earlier

Comment (tkotosz) [reply src/_base/harness/scripts/enable.sh.twig:76]: ah I see...

**Pull request merged**